app/controllers/users_controller.rb

```diff
@@ -1,9 +1,6 @@
 class UsersController < ApplicationController
   before_action :set_user, only: %i[ show liked feed followers following discover ]
-  before_action :authorize_user, except: %i[ index new create ]
-
-  # after_action :verify_authorized, except: :index
-  # after_action :verify_policy_scoped, only: :index
+  before_action -> { authorize @user || User }
```

Comment:
Awesome work using concise authorization for users! Now you can actually take out almost all of your actions in this controller!